wasm-geopolars/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "wasm-geopolars"
+version = "0.1.0"
+authors = ["Kyle Barron <kylebarron2@gmail.com>", "Stuart Lynn"]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+wasm-bindgen = "0.2.63"
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
+
+# `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
+# compared to the default allocator's ~10K. It is slower than the default
+# allocator, however.
+#
+# Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
+wee_alloc = { version = "0.4.5", optional = true }
+
+geopolars = { path = "../geopolars", default-features = false }
+getrandom = { version = "0.2", features = ['js'] }
+
+[dev-dependencies]
+wasm-bindgen-test = "0.3.13"
+
+[profile.release]
+# Tell `rustc` to optimize for small code size.
+opt-level = "s"
+
+[patch.crates-io]
+polars = { git = "https://github.com/stuartlynn/polars.git", version = "0.22.6", branch = 'wasm_rebase', default-features = false, features = [
```

Comment:
Ideally we'd be able to get this to compile _only_ by overriding dependencies here in the `patch` section